modules/ROOT/pages/db-util.adoc

```diff
@@ -136,7 +139,7 @@ The DB server utility has two processors to interact with your DB server:
 
 The MUnit DB Server allows you to validate that the results of a query are as expected.
 
-To do this, you use the `validate-that` operation. Set the `results` property to CSV with rows separated by a newline character (`\n`), as shown below.
+To do this, you use the `validate-that` operation. Set the `results` property to CSV with rows separated by a newline character (`\n`), as follows:
```

Comment:
I'd remove the previous sentence (_The MUnit DB Server allows..._) and unify everything as suggested. 
```suggestion
To validate that the results of a query are as expected, use the `validate-that` operation. Set the `results` property to CSV with rows separated by a newline character (`\n`), as follows:
```